pkgs/unified_analytics/pubspec.yaml

```diff
@@ -14,7 +14,7 @@ dependencies:
   clock: ^1.1.1
   file: ^6.1.4
   http: ^0.13.5
-  intl: ^0.17.0
+  intl: ^0.18.0
```

Comment:
We should also rev the package version for this change; if we intend to publish immediately, we can rev to `0.1.1`; if we want to batch this up w/ future changes, then `0.1.1-dev` (and we should update the changelog at the same time).

Reply:
This is our best doc on how to rev package versions (major / minor / patch): https://github.com/dart-lang/sdk/wiki/External-Package-Maintenance.